[2024-02-03T04:00:35.903+0400] {processor.py:161} INFO - Started process (PID=5763) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:00:35.903+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:00:35.904+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:00:35.904+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:00:35.906+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:00:35.906+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:00:35.906+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:00:35.906+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:00:35.907+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:00:35.913+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-02-03T04:01:29.004+0400] {processor.py:161} INFO - Started process (PID=5987) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:01:29.011+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:01:29.012+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:01:29.012+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:01:29.015+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:01:29.014+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:01:29.015+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:01:29.015+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:01:29.015+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:01:29.024+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-02-03T04:02:19.723+0400] {processor.py:161} INFO - Started process (PID=6202) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:02:19.724+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:02:19.725+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:02:19.725+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:02:19.727+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:02:19.727+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:02:19.728+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:02:19.727+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:02:19.728+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:02:19.735+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-02-03T04:03:14.047+0400] {processor.py:161} INFO - Started process (PID=6443) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:03:14.049+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:03:14.049+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:03:14.049+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:03:14.052+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:03:14.051+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:03:14.052+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:03:14.052+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:03:14.052+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:03:14.060+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-02-03T04:04:07.455+0400] {processor.py:161} INFO - Started process (PID=6669) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:04:07.456+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:04:07.457+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:04:07.456+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:04:07.459+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:04:07.458+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:04:07.459+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:04:07.459+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:04:07.459+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:04:07.466+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-02-03T04:04:55.702+0400] {processor.py:161} INFO - Started process (PID=6873) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:04:55.710+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:04:55.711+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:04:55.710+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:04:55.713+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:04:55.713+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:04:55.714+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:04:55.714+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:04:55.714+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:04:55.721+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-02-03T04:05:53.348+0400] {processor.py:161} INFO - Started process (PID=7139) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:05:53.349+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:05:53.349+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:05:53.349+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:05:53.351+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:05:53.351+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:05:53.352+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:05:53.352+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:05:53.352+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:05:53.358+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-02-03T04:06:41.511+0400] {processor.py:161} INFO - Started process (PID=7364) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:06:41.518+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:06:41.518+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:06:41.518+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:06:41.520+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:06:41.520+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:06:41.520+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:06:41.520+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:06:41.521+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:06:41.527+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-02-03T04:07:38.265+0400] {processor.py:161} INFO - Started process (PID=7623) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:07:38.272+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:07:38.273+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:07:38.273+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:07:38.275+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:07:38.274+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:07:38.275+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:07:38.275+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:07:38.275+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:07:38.281+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-02-03T04:08:27.759+0400] {processor.py:161} INFO - Started process (PID=7838) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:08:27.766+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:08:27.767+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:08:27.766+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:08:27.769+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:08:27.769+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:08:27.769+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:08:27.769+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:08:27.769+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:08:27.776+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-02-03T04:09:18.421+0400] {processor.py:161} INFO - Started process (PID=8054) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:09:18.422+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:09:18.423+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:09:18.423+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:09:18.425+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:09:18.425+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:09:18.425+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:09:18.425+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:09:18.426+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:09:18.432+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-02-03T04:10:09.508+0400] {processor.py:161} INFO - Started process (PID=8500) to work on /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:10:09.510+0400] {processor.py:830} INFO - Processing file /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-02-03T04:10:09.511+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:10:09.511+0400] {dagbag.py:538} INFO - Filling up the DagBag from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:10:09.513+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:10:09.513+0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-02-03T04:10:09.513+0400] {logging_mixin.py:188} INFO - [2024-02-03T04:10:09.513+0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-02-03T04:10:09.514+0400] {processor.py:842} WARNING - No viable dags retrieved from /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-02-03T04:10:09.519+0400] {processor.py:183} INFO - Processing /home/reurairin/projects/urfu/urfu_3s_mlops-3/env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
